### DISCUSSION TITLE
Documentation: link to GitHub Discussions

### DISCUSSION THREAD

**Discussion by Jinksi** [Lightning-AI/pytorch-lightning#7536] [General]
👋 Hi PyTorch Lightning community!

As a newcomer to PyTorch Lightning, I feel that the community on here (Discussions) is really the place to be to connect with each other and get help if needed.

In the docs, there is a link to **Developer Resources** that is ⚠️currently broken, it was linking to **Community Examples**.

![Screen Shot 2021-05-14 at 10 33 55 am](https://user-images.githubusercontent.com/3147296/118204362-65778c80-b4a1-11eb-932f-21419654a3eb.png)

💡 I propose that this link could instead point to GitHub Discussions to help people "Find resources and get questions answers"

I'd be glad to submit a PR with this change if suitable.

**Comment by awaelchli** [comment]
Yes please! I like this idea!
